README.md

```diff
@@ -53,7 +53,7 @@ Build A school library in RUBY using OOP
 - **Add relationships or association to  classroom and student classes and rental, book and person classes**
```

Comment:
- [ ] @clin2on3mun and @olanikegloria Kindly ensure you have the correct title and description for this project in your readme file, currently, you have vet clinic database project as your title.